### ISSUE TITLE
can not start the wallet. Error initializing database..

### ISSUE THREAD

**Issue by BubbaBubbus** [scashml/scashcpp#65]
Hello,
i can not run the wallet on my office pc. it works fine on my notebook. maybe it's the letter **Ü** of the username. (C: \ user \ B**ü**ro-Pc \ AppData ....)

![scs-wallet-fehler](https://user-images.githubusercontent.com/25355054/39545532-c0233a1c-4e51-11e8-9eb7-60f99ce9aab4.jpg)

**Comment by dragononcrypto** [comment]
Yes, very likely this is the issue in my opinion. Using Terminal, there is the option to change your data directory, therefore you could change it to -datadir=~/AppData/Roaming/Scash. This should bypass your username problem. You should be able to include this option in the desktop launcher too.
